tower-balance/src/load/pending_requests.rs

```diff
@@ -1,137 +1,149 @@
-use futures::{Future, Poll, Async};
+use futures::{Async, Poll};
+use std::marker::PhantomData;
 use std::sync::Arc;
-use std::sync::atomic::{AtomicUsize, Ordering};
 use tower_discover::{Change, Discover};
 use tower_service::Service;
 
 use Load;
+use super::{Measure, MeasureFuture, NoMeasure};
 
 /// Expresses load based on the number of currently-pending requests.
-#[derive(Debug, Clone)]
-pub struct PendingRequests<T> {
-    inner: T,
-    pending: Arc<AtomicUsize>,
+#[derive(Debug)]
+pub struct PendingRequests<S, M = NoMeasure>
+where
+    S: Service,
+    M: Measure<Instrument, S::Response>,
+{
+    service: S,
+    track: Arc<()>,
```

Comment:
This is a clever use of `Arc`, but I think a comment explaining what this field is doing might be worth adding, since it's a kind of weird type.